cfme/common/provider_views.py

```diff
@@ -218,7 +218,35 @@ class ProviderRegisterNodesView(View):
      represents Register Nodes view (exists for Infra OpenStack provider)
     """
     file = FileInput(locator='//input[@id="nodes_json_file"]')
-    register = Button('Register')
+    register = Button(value='Register')
+    cancel = Button(value='Cancel')
+
+    @property
+    def is_displayed(self):
+        return False
+
+
+class ProviderScaleDownView(View):
+    """
+     represents Scale down view (exists for Infra OpenStack provider)
```

Comment:
**REQUIRED**
If it exists only in Infra OpenStack provider might be better to place this view into `openstack_infra.py`?